<?xml version="1.0" encoding="UTF-8"?>
<robot generated="20181127 21:14:44.558" generator="Robot 3.0.2 (Python 2.7.15 on win32)">
<suite source="C:\D\plc_system_test" id="s1" name="Plc System Test">
<suite source="C:\D\plc_system_test\tc.4.robot" id="s1-s1" name="Tc.4">
<test id="s1-s1-t1" name="tc_4.1">
<kw type="setup" name="Init Test" library="plc_tb_ctrl">
<msg timestamp="20181127 21:14:44.745" level="INFO">start detail logging</msg>
<msg timestamp="20181127 21:14:44.761" level="FAIL">close relay fail</msg>
<msg timestamp="20181127 21:14:44.761" level="DEBUG">Traceback (most recent call last):
  File "C:\D\plc_system_test\lib\plc_tb_ctrl.py", line 218, in init_test
    TB_INSTANCE.set_event_low()
  File "C:\D\plc_system_test\lib\plc_tb_ctrl.py", line 1214, in set_event_low
    assert 0 == result, "close relay fail"</msg>
<status status="FAIL" endtime="20181127 21:14:44.761" starttime="20181127 21:14:44.745"></status>
</kw>
<kw type="teardown" name="Record Test Status" library="plc_tb_ctrl">
<msg timestamp="20181127 21:14:44.761" level="INFO">begin 5 seconds waiting before exit...</msg>
<msg timestamp="20181127 21:14:50.761" level="INFO">stop detail logging</msg>
<status status="PASS" endtime="20181127 21:14:50.777" starttime="20181127 21:14:44.761"></status>
</kw>
<status status="FAIL" endtime="20181127 21:14:50.777" critical="yes" starttime="20181127 21:14:44.745">Setup failed:
close relay fail</status>
</test>
<status status="FAIL" endtime="20181127 21:14:50.777" starttime="20181127 21:14:44.698"></status>
</suite>
<status status="FAIL" endtime="20181127 21:14:50.777" starttime="20181127 21:14:44.683"></status>
</suite>
<statistics>
<total>
<stat fail="1" pass="0">Critical Tests</stat>
<stat fail="1" pass="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat fail="1" id="s1" name="Plc System Test" pass="0">Plc System Test</stat>
<stat fail="1" id="s1-s1" name="Tc.4" pass="0">Plc System Test.Tc.4</stat>
</suite>
</statistics>
<errors>
</errors>
</robot>
